compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -784,6 +787,26 @@ class JSExportTest extends DirectTest with TestHelpers {
   }
 
   @Test
+  def namedExportIsDeprecated: Unit = {
+
+    """
+    class A {
+      @JSExportNamed
+      def foo(x: Int, y: Int) = 1
+    }
+    """ hasWarns
+    """
+      |newSource1.scala:5: warning: class JSExportNamed in package annotation is deprecated: Use @JSExport with an explicit option bag instead. See the Scaladoc for more details.
+      |      def foo(x: Int, y: Int) = 1
+      |          ^
+      |newSource1.scala:4: warning: class JSExportNamed in package annotation is deprecated: Use @JSExport with an explicit option bag instead. See the Scaladoc for more details.
+      |      @JSExportNamed
+      |       ^
```

Comment:
The duplicate warning is due to a weird interaction with `PreJSExports`, which creates an export bridge *with* the same annotation. Apparently deprecation warnings are handled later in the pipeline, and so the deprecation is reported twice.

TBH, I don't care ^^